src/CssInlinerPlugin.php

```diff
@@ -48,6 +52,30 @@ public function handleSymfonyEvent(MessageEvent $event): void
         $this->handleSymfonyEmail($message);
     }
 
+    private function buildCssContent(): void
+    {
+        $this->addCssFromFiles();
+        $this->addCssFromContent();
+    }
+
+    private function addCssFromFiles(): void
+    {
+        $filesToInline = $this->config['css-files'] ?? [];
+
+        $this->cssToAlwaysInclude = $this->loadCssFromFiles($filesToInline);
```

Comment:
If both functions are used to add to the `$this->cssToAlwaysInclude` perhaps it would be best to also use the `.=` here as then the order of method calling has no side effects.
```suggestion
        $this->cssToAlwaysInclude .= $this->loadCssFromFiles($filesToInline);
```